resource-timing/resources/all_resource_types.htm

```diff
@@ -6,6 +6,35 @@
         <link rel="shortcut icon" href="empty.py?favicon">
     </head>
     <body background='blue.png?id=body'>
+        <script>
+            let count = 0;
+            let totalEventSourceCount = 0;
+
+            function perf_observer(list, observer) {
+                const entries = list.getEntries();
+                const eventSourceCount = entries.filter(e => e.name.includes('eventsource')).length;
+
+                if(eventSource > 0) {
+                    totalEventSourceCount += eventSourceCount;
+                } 
+                
+                // exclude eventSource since it arrives repeatedly
+                count += entries.filter(e => !e.name.includes('eventsource')).length;
+                
+                if(count === 34 && totalEventSourceCount > 0) {
+                    // we are done only if all static resources arrived and at least one event source
+                    observer.disconnect();
+                    parent.on_all_resources_loaded();
```

Comment:
Should this check that it has the property, like `on_test_body_created`?

Reply:
it is not needed since that is checked outside in before the final verification in `resource-timing/resource_initiator_types.html`